src/hidra/sender/eventdetectors/watchdog_events.py

```diff
@@ -489,7 +503,8 @@ def check_last_modified(self, filepath):
             return
 
         # compare ( >= limit)
-        if time_current - time_last_modified >= self.time_till_closed:
+        # file_size_check > 0 Check is introduced to stop the file transfer when they have 0kb size 
```

Comment:
```suggestion
        # In rare cases, empty files can stay open without modifications for much longer than usual.
        # To avoid setting `time_till_closed` to a usually unnecessary long time, empty files are only considered closed
        # after 10 times the configured `time_till_closed`.
```

Reply:
Fixed.